mtags-interfaces/src/main/java/scala/meta/pc/PresentationCompiler.java

```diff
@@ -141,7 +142,7 @@ public abstract CompletableFuture<List<AutoImportsResult>> autoImports(String na
 	/**
 	 * Return named arguments for the apply method that encloses the given position.
 	 */
-	public abstract CompletableFuture<List<TextEdit>> convertToNamedArguments(OffsetParams params,
+	public abstract CompletableFuture<Either<String, List<TextEdit>>> convertToNamedArguments(OffsetParams params,
```

Comment:
Could we instead of using an either complete the future with an exception? We could handle the exception when invoking `handleCommand` in MetalsLspService  around line 2068 ?

```
        CancelTokens.future { token =>
          codeActionProvider.executeCommands(params, token).recover{
            case exception =>
              languageClient.showMessage(l.MessageType.Info, exception.getMessage)
          }.withObjectValue
        }
```